framework/src/Volo.Abp.Ddd.Domain/Volo/Abp/Domain/Entities/EntityHelper.cs

```diff
@@ -266,5 +266,36 @@ public static Type FindPrimaryKeyType([NotNull] Type entityType)
 
             property?.SetValue(entity, idFactory());
         }
+
+        public static void TrySetId<TKey>(
+            object entity,
+            TKey id,
+            bool checkForDisableIdGenerationAttribute = false)
+        {
```

Comment:
The method above can internally use this one to avoid code duplication. Becuase `IEntity<TKey>` is already an object.